src/BigCommerce/ResourceModels/Catalog/Product/PriceAdjuster.php

```diff
@@ -11,4 +11,13 @@ class PriceAdjuster extends ResourceModel
 
     public string $adjuster;
     public float $adjuster_value;
+    
+    protected function beforeBuildObject(): void
+    {
+        if (!is_null($this->optionObject)) {
+            if(is_null($this->optionObject->adjuster)){
+                $this->optionObject->adjuster = self::ADJUSTER_FIXED;
```

Comment:
Is this to handle the case of BigCommerce not returning the `adjuster` property?

Reply:
Yes, this will handle when BigCommerce is not returning adjuster property.

Reply:
It is usally not returning adjuster property when in store front we have fixed type of adjuster.